File: codechef/February Challenge 2021 Division 3/HDIVISR.cpp
// link: https://www.codechef.com/FEB21C/problems/HDIVISR
// contest code: https://www.codechef.com/FEB21C?order=desc&sortBy=successful_submissions
#include<iostream>
int main(){
    int N;
    int res=0;
    std::cin>>N;
    for(int i=1; i<=10; i++){
        if(N%i == 0){
            res = i;
        }
    }
    std::cout<<res;
    return 0;
}

File: codechef/February Challenge 2021 Division 3/HDIVISR.py
# link: https://www.codechef.com/FEB21C/problems/HDIVISR
N = int(input())    # 2≤N≤1,000
res = 0
for a in range(9):
    if N%(a+1) == 0:
        res = (a+1)
print(res)